### PR TITLE
Adding Deep-links from README to code snippets

### PR DESCRIPTION
**Pull request by iambmelt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
...
-* Send a message from the user.
+* [Get user's messages from Office 365.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/MessageSnippets.java#L46)
+* [Send a message from the user.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/MessageSnippets.java#L58)
 
 **Contacts**
 
-* Get all contacts in the tenant.
-* Create a new contact in your tenant.
+* [Get all contacts in the tenant.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/ContactsSnippets.java#L37)
+* [Create a new contact in your tenant.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/UsersSnippets.java#L68)
```
Comment (davechmsft) [inline README.md]: create contact is not in the code and should be removed from the readme.

File: README.md [hunk 2]
```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
...
-* Get all groups in your tenant's directory.
-* Create a new security group.
-* Get information about a specific group in the tenant.
-* Update the description of a group.
-* Delete a group.
-* Get a group's members.
-* Get a group's owners.
+* [Get all groups in your tenant's directory.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L138)
+* [Create a new security group.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L152)
+* [Get information about a specific group in the tenant.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L57)
```
Comment (davechmsft) [inline README.md]: This link needs to be 2 code lines higher

File: README.md [hunk 3]
```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
...
+* [Update the description of a group.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L170)
```
Comment (davechmsft) [inline README.md]: This link needs to be 2 code lines higher

File: README.md [hunk 4]
```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
...
+* [Delete a group.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L197)
+* [Get a group's members.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L93)
```
Comment (davechmsft) [inline README.md]: This link needs to go higher to line 84

File: README.md [hunk 5]
```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
...
+* [Get a group's owners.](app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/GroupsSnippets.java#L122)
```
Comment (davechmsft) [inline README.md]: This link should go to line 113.

File: README.md [hunk 6]
```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
...
 
 **Drives**
 
-* Gets the signed-in user's drive.
-* Gets  all of the drives in your tenant.
+* [Gets the signed-in user's drive.](/app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/DrivesSnippets.java#L55)
```
Comment (davechmsft) [inline README.md]: 2 lines higher

File: README.md [hunk 7]
```diff
@@ -20,50 +20,58 @@ You can explore the following operations for Office 365:
...
+* [Gets  all of the drives in your tenant.](/app/src/main/java/com/microsoft/office365/unifiedsnippetapp/snippet/DrivesSnippets.java#L68)
```
Comment (davechmsft) [inline README.md]: I think there are several of these that don't link to the top of the snippet, so I'll stop commenting on them here. :-) I would just say they should all link to the name of the snippet, like get_organization_drives and such.

Comment (davechmsft) [inline README.md]: actually the rest are correct!

**Comment by iambmelt** [comment]
Thanks for the code review @davechmsft!

Hey @JohnAustin-MSFT - do you have time to take care of these or should I?

**Comment by JohnAustin-MSFT** [comment]
DONE.

File: README.md (hunk 1 above)
Comment (iambmelt) [inline README.md]: resolved :+1:

File: README.md (hunk 2 above)
Comment (iambmelt) [inline README.md]: Resolved :+1:

**Pull request merged**